qiskit/algorithms/linear_solvers/numpy_linear_solver.py

```diff
@@ -0,0 +1,97 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020, 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""The Numpy LinearSolver algorithm (classical)."""
+
+from typing import List, Union, Optional, Callable
+import numpy as np
+
+from qiskit import QuantumCircuit
+from qiskit.quantum_info import Operator, Statevector
+from qiskit.quantum_info.operators.base_operator import BaseOperator
+
+from .linear_solver import LinearSolverResult, LinearSolver
+from .observables.linear_system_observable import LinearSystemObservable
+
+
+class NumPyLinearSolver(LinearSolver):
+    """
+    The Numpy LinearSolver algorithm (classical).
+
+    This linear system solver computes the exact value of the given observable(s) or the full
+    solution vector if no observable is specified.
+
+    Examples:
+
+        .. jupyter-execute::
+
+            import numpy as np
+            from qiskit.algorithms.numpy_linear_solver import NumPyLinearSolver
+            from qiskit.algorithms.linear_solvers.matrices.tridiagonal_toeplitz import
+             TridiagonalToeplitz
+            from qiskit.algorithms.linear_solvers.observables.matrix_functional import
+             MatrixFunctional
```

Comment:
These are syntax errors and why the docs builds are failing if you need a new line here (I assume because it's >100 characters) you can use `\` to fix this. For example something like:

```suggestion
            from qiskit.algorithms.linear_solvers.matrices.tridiagonal_toeplitz import \
             TridiagonalToeplitz
            from qiskit.algorithms.linear_solvers.observables.matrix_functional import \
             MatrixFunctional
```

should work (although I haven't tested that so it might have whitespace issues or something)

Reply:
https://dev.azure.com/qiskit-ci/qiskit-terra/_build/results?buildId=26226&view=logs&j=e32fad43-1785-5ab1-d2e9-2c0ab2cb49da&t=10db491d-7a76-589f-3b30-e8d05469d998&l=125 is the docs failure